CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Beta 0.7.0
```

Comment:
Can you leave the old `Beta 0.7.0` alone and create a new `Beta 0.7.1` section with a comment describing your change, since 0.7.0 is already released.

Reply:
Added. Thanks for that.